### ISSUE TITLE
two questions about timer

### ISSUE THREAD

**Issue by zxl696** [thuml/Large-Time-Series-Model#27]
1.                if self.args.use_ims:
                    # output used to calculate loss misaligned patch_len compared to input
                    loss = criterion(outputs[:, -self.args.seq_len:, :], batch_y)
                else:
                    # only use the forecast window to calculate loss
                    loss = criterion(outputs[:, -self.args.pred_len:, :], batch_y[:, -self.args.pred_len:, :])
代码这里的计算损失为什么是这样的，使用ims为啥这样计算损失，训练函数里面也没看出来使用ims他的训练逻辑
2.        if self.data_type == 'custom':
            data_len = len(df_raw)
            num_train = int(data_len * 0.75)
            num_test = int(data_len * 0.15)
            num_vali = data_len - num_train - num_test
            border1s = [0, num_train-self.input_len, data_len - num_test-self.input_len]
            border2s = [num_train, num_train + num_vali, data_len]
这样分配起始点会不会造成测试集和验证集提前知道数据了

**Comment by WenWeiTHU** [comment]
Hi, please refer to the two relevant issue:
* https://github.com/thuml/Large-Time-Series-Model/issues/24
* https://github.com/thuml/Autoformer/issues/53
* https://github.com/thuml/Autoformer/issues/20
